translations/catroweb.en.yml

```diff
@@ -184,6 +184,9 @@ programs:
   link: Copy project link to clipboard
   reactionsText: Reactions
   downloadErrorText: Error occurred while downloading the project. Please try again later.
+  showTranslation: Show Translation
+  hideTranslation: Hide Translation
+  provider: Translated by %provider% from %sourceLanguage% to %targetLanguage%
```

Comment:
They might want to use snake_case here for the placeholders, but we'll see

Reply:
also rename maybe to `translatedByText` instead of `provider`

Reply:
Renamed provider to translatedByText